### DISCUSSION TITLE
What is the Syntax to Switch LoRA Off or On in Preset or Config.txt?

### DISCUSSION THREAD

**Discussion by DavidDragonsage** [lllyasviel/Fooocus#2479] [Q&A] [answered]
I really appreciate the Fooocus platform, and am happy with the new features in version 2.2. Thank you to the team for all your dedication and hard work!

I would like to control whether a LoRA starts off being enabled or disabled (checked or unchecked).

**Comment by mashb1t** [comment]
This currently is not possible, you can either set a LoRA or set None.
Feel free to implement it and open a PR!

**Reply by DavidDragonsage** [reply]
I'm afraid I would be out of my depth on that one! I somehow got the impression there was a way to preset this value, but of course I couldn't find the procedure. Thank you for the prompt reply! :)

**Comment by lgwjames** [comment]
just untick the lora on the box on the left under model?

**Reply by mashb1t** [reply]
The idea behind the request is to show some LoRAs by default, but don't activate them. This enables users to have an even lower barrier of entry.

**Reply by DavidDragonsage** [reply]
Yes, that's exactly what I was thinking, I have presets set up for my wife to use, who is more of an artist than a technologist. I would like to include useful LoRAs in those presets, set for the appropriate weights, but defaulting to Off until she wants them. On the other hand there are some LoRAs that she would always want to be On. So it would be really useful to have the ability to set the default On/Off state for each LoRA within the preset.

A workaround would be to have the last two LoRAs always default to Off, and the first two LoRAs to On. That might require less programming and I would think that in most cases that would solve this issue.

**Reply by mashb1t** [reply]
now in issue / feature request https://github.com/lllyasviel/Fooocus/issues/2503

**Reply by mashb1t** [reply]
implemented in https://github.com/lllyasviel/Fooocus/pull/2507

**Comment by lgwjames** [comment]
so in my config i have

 "default_loras": [
        [
            "JessicaL.safetensors",
            1.0
        ],
        [
            "PerfectEyesXL.safetensors",
            1.0
        ],
        [
            "Perfect Hands v2.safetensors",
            1.0
        ],
        [
            "None",
            1.0
        ],
        [
            "None",
            1.0
        ]
],

maybe a bit of code like  "default_advanced_checkbox": true,

**Reply by DavidDragonsage** [reply]
Yes indeed.

**Comment by mashb1t** [comment, marked as answer]
https://github.com/lllyasviel/Fooocus/pull/2507 is now merged to develop and will be included in the next version of Fooocus.
